lib/bootstrap.js

```diff
@@ -5,7 +5,7 @@ const cmdArgs = require('command-line-args');
 require('./utils/process-log');
 
 const isFlagPresent = (args, flag) => args.find(arg => [flag, `--${flag}`].includes(arg));
-const normalizeFlags = (args, command) => args.slice(2).filter(arg => arg.indexOf('--') < 0 && arg !== command.name && arg !== command.alias);
+const normalizeFlags = (args, cmd) => args.slice(2).filter(arg => arg.indexOf('--') < 0 && arg !== cmd.name && arg !== cmd.alias);
```

Comment:
IMO changing `command` to `cmd` doesn't bring much of a value to the meaningfulness of the variable nor to the readability of code.

Reply:
Just to stay consistent with rest of the code-base.
https://github.com/webpack/webpack-cli/blob/8a54b58e68d976c4265c0d466854d69f779a6650/lib/bootstrap.js#L11

Reply:
https://github.com/webpack/webpack-cli/blob/8a54b58e68d976c4265c0d466854d69f779a6650/lib/bootstrap.js#L12